exercises/practice/dnd-character/.meta/example.rb

```diff
@@ -0,0 +1,53 @@
+module Die
+  extend self
+
+  def roll(roll: 4, die: 6, remove_lowest: 1)
+    roll.times.map { rand(1..die) }.sort.pop(roll - remove_lowest).sum
+  end
+
+end
+
+module Modifiable
+
+  # This only calculates hit point modifier based on constitution
+  def modifier(constitution)
+    constitution/2 - 5
+  end
+
+end
+
+class DndCharacter
+  extend Modifiable
+
+  BASE_HITPOINTS = 10
+
+  private_constant :BASE_HITPOINTS
+
+  private
+
+  def initialize
+    @strength     = Die.roll
+    @dexterity    = Die.roll
+    @constitution = Die.roll
+    @intelligence = Die.roll
+    @wisdom       = Die.roll
+    @charisma     = Die.roll
+
+    @hitpoints    = BASE_HITPOINTS + self.class.modifier(constitution)
```

Comment:
And give, as it is here, the more specific thing.

Reply:
Using `ability_score` as the new argument based of 5e naming conventions: https://roll20.net/compendium/dnd5e/Ability%20Scores#content

Reply:
Wasn't sure what you were asking for here, the modifier method takes an ability score, and in this case we want to calculate it from constitution, so I don't believe this needs any naming changes based on the use case.

Reply:
I may have been looking at some other version, you are correct.